ecs-agent/api/ecs/client/ecs_client.go

```diff
@@ -519,7 +524,7 @@ func (client *ecsClient) submitTaskStateChange(change ecs.TaskStateChange) error
 		}
 
 		_, err := client.submitStateChangeClient.SubmitTaskStateChange(&ecsmodel.SubmitTaskStateChangeInput{
```

Comment:
nit: Let's look into adding a unit test for STSC that has as input a `TaskStateChange` with `ClusterARN` field populated. I understand that there is a level of urgency to merging the underlying change of this pull request, so this can be done in a followup pull request if not done in this one.

Reply:
Sure.